include/bluetooth/audio/capabilities.h

```diff
@@ -0,0 +1,260 @@
+
+	/** Preferred PHY */
+	uint8_t phy;
+
+	/** Preferred Retransmission Number */
+	uint8_t rtn;
+
+	/** Preferred Transport Latency in ms */
+	uint16_t latency;
+
+	/** @brief Minimun Presentation Delay in us
+	 *
+	 *  Unlike the other fields, this is not a preference but a minimum
+	 *  requirement.
+	 */
+	uint32_t pd_min;
+
+	/** @brief Maximum Presentation Delay in us
+	 *
+	 *  Unlike the other fields, this is not a preference but a maximum
+	 *  requirement.
+	 */
+	uint32_t pd_max;
+
+	/** @brief Preferred minimun Presentation Delay in us*/
+	uint32_t pref_pd_min;
+
+	/** @brief Preferred maximum Presentation Delay	in us */
+	uint32_t pref_pd_max;
+};
+
+struct bt_audio_capability; /* Handle circular dependency */
+
+/** @brief Capability operations structure.
+ *
+ *  These are only used for unicast streams and broadcast sink streams.
+ */
+struct bt_audio_capability_ops {
+	/** @brief Capability config callback
+	 *
+	 *  Config callback is called whenever a new Audio Stream needs to be
+	 *  allocated.
+	 *
+	 *  @param conn Connection object
+	 *  @param ep Remote Audio Endpoint being configured
+	 *  @param cap Local Audio Capability being configured
+	 *  @param codec Codec configuration
+	 *
+	 *  @return Allocated Audio Stream object or NULL in case of error.
+	 */
+	struct bt_audio_stream *(*config)(struct bt_conn *conn,
+					  struct bt_audio_ep *ep,
+					  struct bt_audio_capability *cap,
+					  struct bt_codec *codec);
+
+	/** @brief Capability reconfig callback
+	 *
+	 *  Reconfig callback is called whenever an Audio Stream needs to be
+	 *  reconfigured with different codec configuration.
+	 *
+	 *  @param stream Stream object being reconfigured.
+	 *  @param cap Local Audio Capability being reconfigured
+	 *  @param codec Codec configuration
+	 *
+	 *  @return 0 in case of success or negative value in case of error.
+	 */
+	int (*reconfig)(struct bt_audio_stream *stream,
+			struct bt_audio_capability *cap,
+			struct bt_codec *codec);
+
+	/** @brief Capability QoS callback
+	 *
+	 *  QoS callback is called whenever an Audio Stream Quality of
+	 *  Service needs to be configured.
+	 *
+	 *  @param stream Stream object being reconfigured.
+	 *  @param QoS Quality of Service configuration
+	 *
+	 *  @return 0 in case of success or negative value in case of error.
+	 */
+	int (*qos)(struct bt_audio_stream *stream, struct bt_codec_qos *qos);
+
+	/** @brief Capability Enable callback
+	 *
+	 *  Enable callback is called whenever an Audio Stream is about to be
+	 *  enabled to stream.
+	 *
+	 *  @param stream Stream object being enabled.
+	 *  @param meta_count Number of metadata entries
+	 *  @param meta Metadata entries
+	 *
+	 *  @return 0 in case of success or negative value in case of error.
+	 */
+	int (*enable)(struct bt_audio_stream *stream, uint8_t meta_count,
+		      struct bt_codec_data *meta);
+
+	/** @brief Capability Start callback
+	 *
+	 *  Start callback is called whenever an Audio Stream is about to be
```

Comment:
"to start streaming" or "to be started streaming"

Reply:
Fixed this and a few similar oddities in the descriptions.